HZN-one/flex
# #204 Molekul - FMCard

## Thanks for your support to the project. Please check below mentioned points -

### Detailed description of changes

- Need to update Default Padding Card to 24px, 
- Need to update Card border radius to 8px


### Recommendations on how to test the changes

- Check for the card padding
- Check for the card border radius
- Can customized using `sx` if required to do so

### Checklist for developer


### Link to related issue(s)

https://app.zenhub.com/workspaces/flick-614155581501e1000f356a5d/issues/hzn-one/tms/204

### Screenshots or videos (before and after if appropriate)

#### Before

N/A

#### After

![image](https://user-images.githubusercontent.com/46083126/153841122-b23b9a81-36c1-4082-80f5-f2e256adf40e.png)
